vendor/github.com/BurntSushi/toml/.travis.yml

```diff
@@ -2,11 +2,14 @@ language: go
 go:
   - 1.1
   - 1.2
+  - 1.3
+  - 1.4
+  - 1.5
+  - 1.6
```

Comment:
Are u telling me that it executes the travis.yml of all the go dependencies :D ?